doc/xml/reference.xml

```diff
@@ -1062,6 +1062,17 @@
                         <example>1</example>
                     </config-key>
 
+                    <!-- ======================================================================================================= -->
+                    <config-key id="pg-database" name="PostgreSQL Database">
+                        <summary><postgres/> database user.</summary>
```

Comment:
Why is the summary about the database user?

Reply:
Fixed, as below.